weave/weave_client.py

```diff
@@ -487,9 +487,16 @@ def finish_call(
         summary = {}
         if call._children:
             summary = sum_dict_leaves([child.summary or {} for child in call._children])
-        elif isinstance(output, dict) and "usage" in output and "model" in output:
+        elif (
```

Comment:
we want to use `original_output` here so that we don't accidentally start operating on refs